app/views/courses/EnrollmentsView.coffee

```diff
@@ -74,6 +74,8 @@ module.exports = class EnrollmentsView extends RootView
     @debouncedRender = _.debounce @render, 0
     @listenTo @prepaids, 'sync', @updatePrepaidGroups
     @listenTo(@state, 'all', @debouncedRender)
+
+    me.getClientCreatorPermissions()?.then(() => @render?())
```

Comment:
Are all these single calls added to views for `teacher-dashboard-nav.jade`?

Reply:
Yes, to hide the Student Licenses tab.